packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1083,6 +1089,12 @@ mixin TextSelectionDelegate {
   /// If [cause] is [SelectionChangedCause.toolbar], the position of
   /// [bringIntoView] to selection will be called and hide toolbar.
   void copySelection(SelectionChangedCause cause);
+
+  /// Start Live Text input.
+  ///
+  /// If and only if [cause] is [SelectionChangedCause.toolbar], the toolbar
+  /// will be hidden and will start Live Text input.
```

Comment:
This isn't true; Live Text input will be started regardless of what the `cause` is!  The part about the toolbar being hidden is true, though.